server/database/models/AssignedChore.js

```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
+const db = require('./../db');
+
+const AssignedChore = db.define('assignedChores', {
```

Comment:
'assignedChore'

Reply:
good catch, completely spaced over this one.